udfs/udfs.py

```diff
@@ -0,0 +1,28 @@
+import pickle
```

Comment:
Not sure about this file? we already have a UDF architecture. I assume this is just a playground file.

Reply:
Removed. Thanks for pointing this out.

Reply:
Removed. Thanks for pointing this out.